Base.rte/Activities/OneManArmy.lua

```diff
@@ -176,10 +270,10 @@ function OneManArmy:UpdateActivity()
 					DeleteEntity(ship);
 					ship = RandomACRocket("Any", self.CPUTechName);
 				end
-				actorsInCargo = ship.MaxPassengers
+				actorsInCargo = math.min(ship.MaxPassengers, 3);
```

Comment:
It's simply to make the activity easier, you get overwhelmed pretty easily. I also must've had set a higher passenger limit to dropships back then.

Reply:
Okay, if it's gotta stay for balance reasons then so be it.
That said, I'd rather keep this based on passenger limit naturally, and lower spawn rates or increase player buffs or something. Generally speaking, overriding behaviour in weird ways like this can make it very easy to lose track of why you overrode that behaviour (especially if you're looking back at a script in a few months time), and makes it harder to make sense of the whole thing.